deploy/eks/kiam-server-deplyoyment.yaml

```diff
@@ -0,0 +1,113 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: kube-system
+  name: kiam-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kiam
+      role: server
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/port: "9620"
+      labels:
+        app: kiam
+        role: server
+    spec:
+      serviceAccountName: kiam-server
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - topologyKey: kubernetes.io/hostname
+      nodeSelector:
+        kubernetes.io/role: cluster-management
+      tolerations:
```

Comment:
How did you get it work with AWS EKS?

Reply:
Tried out with dedicated node, first time saw some heartbeat issues, second try works.

Reply:
We can not deploy server on EKS master. Agent has to be on different node, my delegate service pod need to be on the agent node. Seems restricted.

Reply:
@lemonshow could you explain how you had this working?